scripts/uberenv/spack_configs/toss_3_x86_64_ib/packages.yaml

```diff
@@ -10,123 +10,134 @@ packages:
+    - spec: automake
+      prefix: /usr
   bzip2:
-   paths:
-      bzip2: /usr
-   buildable: False
-  cmake:
-    version: [3.14.5]
-    paths:
-      cmake: /usr/tce/packages/cmake/cmake-3.14.5
-    buildable: False
+    buildable: false
+    externals:
+    - spec: bzip2
+      prefix: /usr
   gettext:
-   paths:
-      gettext: /usr
-   buildable: False
+    buildable: false
+    externals:
+    - spec: gettext
+      prefix: /usr
+  graphviz:
+    buildable: false
+    externals:
+    - spec: graphviz
+      prefix: /usr
   libtool:
-   paths:
-      libtool: /usr
-   buildable: False
+    buildable: false
+    externals:
+    - spec: libtool
+      prefix: /usr
+  libx11:
+    buildable: false
+    externals:
+    - spec: libx11
+      prefix: /usr
   m4:
-   paths:
-      m4: /usr
-   buildable: False
+    buildable: false
+    externals:
+    - spec: m4
+      prefix: /usr
   perl:
-    paths:
-      perl: /usr
-    buildable: False 
+    buildable: false
+    externals:
+    - spec: perl
+      prefix: /usr
   pkg-config:
-    paths:
-      pkg-config: /usr
-    buildable: False 
+    buildable: false
+    externals:
+    - spec: pkg-config
+      prefix: /usr
   tar:
-   paths:
-      tar: /usr
-   buildable: False
-  graphviz:
-    paths:
-      graphviz: /usr
-    buildable: False
-  libx11:
-    paths:
-      libx11: /usr
-    buildable: False
+    buildable: false
+    externals:
+    - spec: tar
+      prefix: /usr
+  readline:
+    buildable: false
+    externals:
+    - spec: readline
+      prefix: /usr
+  unzip:
+    buildable: false
+    externals:
+    - spec: unzip
+      prefix: /usr
+  zlib:
+    buildable: false
+    externals:
+    - spec: zlib
+      prefix: /usr
 
 # Lock in versions of Devtools
+  cmake:
+    version: [3.14.5]
+    buildable: false
+    externals:
+    - spec: cmake
+      prefix: /usr/tce/packages/cmake/cmake-3.14.5
   cppcheck:
     version: [1.87]
-    paths:
-      cppcheck: /usr/WS2/smithdev/devtools/toss_3_x86_64_ib/latest/cppcheck-1.87
-    buildable: False
+    buildable: false
+    externals:
+    - spec: cppcheck
+      prefix: /usr/WS2/smithdev/devtools/toss_3_x86_64_ib/latest/cppcheck-1.87
   doxygen:
     version: [1.8.17]
-    paths:
-      doxygen: /usr/WS2/smithdev/devtools/toss_3_x86_64_ib/latest/doxygen-1.8.17
-    buildable: False
+    buildable: false
+    externals:
+    - spec: doxygen
+      prefix: /usr/WS2/smithdev/devtools/toss_3_x86_64_ib/latest/doxygen-1.8.17
   python:
-    version: [3.7.7]
-    paths:
-      python: /usr/WS2/smithdev/devtools/toss_3_x86_64_ib/latest/python-3.7.7
-    buildable: False
+    buildable: false
+    externals:
+    - spec: python@3.8.2
+      prefix: /usr/tce/packages/python/python-3.8.2
   py-sphinx:
-    version: [3.0.0]
-    paths:
-      py-sphinx: /usr/WS2/smithdev/devtools/toss_3_x86_64_ib/latest/py-sphinx-3.0.0
-    buildable: False
+    version: [2.2.0]
+    buildable: false
+    externals:
+    - spec: py-sphinx@2.2.0
```

Comment:
I am a bit concerned on the multiple versions sphinx we have now... though i do all my docs building through readthedocs so maybe it doesn't actually matter.